game/scripts/npc/abilities/meepo_poof.txt

```diff
@@ -28,7 +28,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "80"
+    "AbilityManaCost"                                     "60" //OAA
```

Comment:
Meepo changes allowed him to be picked once again but he's still being outclassed due to his relative low farming time (clones don't get many item benefits nor benefits from the sparks) and early game poof is how he would farm. a slight buff to the MC will allow him to maybe exist more in a C or B tier for carries rather than a D tier.

Reply:
seems fine. probably also has minimal impact on meepos position but it can't hurt. power spark isn't super relevant on meepo since you need levels.

Reply:
> Meepo is not outclassed because of the farming speed (Power Spark works on his clones). He is outclassed because he is a hero that requires micro and he has no dmg steroid ability like crit or something. His abilities are so bad, he will never be a carry (because his abilities don't scale with items), unless he is completely reworked. Another reason Meepo isn't played is his (their) abilities - they are boring in this day and age. Anyways, we don't do mana cost changes in this game unless they are at OAA levels or the heroes that have too much mana (Leshrac, Tinker) need a nerf.

he did have a dmg steroid on his E, that you guys hard nerfed for some reason. Either revert that nerf or just let this MC change go through. As the only person who played meepo I will say he's one of hte few heroes who run out of mana naturally

Reply:
1) We never nerfed E
2) You are not the only one playing Meepo.
3) Set the mana cost to 50 if you want to make this relevant.

Reply:
you guys did nerf E, after I had previously buffed it. It may of been chris.

Thoughts on removing manacost altogether?

Reply:
If you think meepo is bad I can just buff him in other ways as well lol

Reply:
0 mana cost would be annoying.

Reply:
the E nerf was prior to the full release. It was changed in the original PR I believe, by you.

Reply:
can't find it in any PR. It may of been simply on my end I just remember the numbers being way higher.

Reply:
think it was like: 10 14 18 24 36 72 or something.

Reply:
ye there was discussion in the DM about it in Jan.